lib/bowermanager.js

```diff
@@ -162,43 +154,44 @@ BowerManager.prototype.installPlugin = function(pluginName, pluginVersion, callb
  */
 BowerManager.prototype.installLatestCompatibleVersion = function (pluginName, callback) {
   var self = this;
-    fs.readJson(path.join(app.configuration.getConfig('root'), 'version.json'), function(error, versionJson) {
-      if (error) {
-        logger.log('error', error);
-        return callback(err);
+  // Query bower to verify that the specified plugin exists.
+  bower.commands.search(pluginName, bowerOptions)
+    .on('error', callback)
+    .on('end', function (results) {
+      if (!results || results.length == 0) {
+        logger.log('warn', 'Plugin ' + packageName + ' not found!');
```

Comment:
Do we need to copy the `packageName` declaration from [here](https://github.com/taylortom/adapt_authoring/blob/d0f63489cbd025c3af2c488f96638018cb340cd7/lib/bowermanager.js#L76-L78)?